stablehlo/tests/interpret_square_root.mlir

```diff
@@ -0,0 +1,25 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: square_root_op_test_f64
+func.func @square_root_op_test_f64() -> tensor<4xf64> {
+  %0 = stablehlo.constant dense<[25.33, -36.11, 1.0, 0.0]> : tensor<4xf64>
+  %1 = stablehlo.sqrt %0 : tensor<4xf64>
+  func.return %1 : tensor<4xf64>
+  // CHECK-NEXT: tensor<4xf64>
+  // CHECK-NEXT: 5.0328918128646478 : f64
+  // CHECK-NEXT: 0xFFF8000000000000 : f64
+  // CHECK-NEXT: 1.000000e+00 : f64
+  // CHECK-NEXT: 0.000000e+00 : f64
+}
+
+// -----
+
+// CHECK-LABEL: Evaluated results of function: square_root_op_test_c64
+func.func @square_root_op_test_c64() -> tensor<2xcomplex<f32>> {
```

Comment:
According to the [testing_guidelines](https://github.com/openxla/stablehlo/blob/main/docs/reference.md#testing-guidelines), we should be testing with the maximum bit-width for each type:
```
func.func @square_root_op_test_c128() -> tensor<2xcomplex<f64>> {
```